test/core/create-node.spec.js

```diff
@@ -19,7 +21,7 @@ describe('create node', function () {
     this.timeout(80 * 1000)
 
     const node = new IPFS({
-      repo: '/tmp/ipfs-repo-' + Math.random(),
+      repo: path.join(os.tmpdir(), 'ipfs-repo-' + Math.random()),
```

Comment:
one more (ref: https://github.com/ipfs/ipfs-performance-profiling/pull/12#discussion_r154862400)

Reply:
@diasdavid Good idea.  I don't like seeing Math.random, because I always worry if a cryptographically secure P/RNG should be used.